src/klein/test/test_resource.py

```diff
@@ -524,13 +524,12 @@ def root(request):
                     os.path.dirname(__file__), "__init__.py"), 'rb').read())
         self.assertEqual(request.finishCount, 1)
 
-
     def test_explicitStaticBranch(self):
         app = self.app
 
         request = requestMock(b"/static/__init__.py")
 
-        @app.route("/static/", branch=True)
+        @app.route("/static", branch=True)
```

Comment:
I'm not really sure why I changed this - that said, I agree that there should be a tests for both `/static` and `/static/` that show that they produce the same result.